plusminus/plusminus.py

```diff
@@ -1016,13 +1034,46 @@ def set_union(a, b):
                 else PrettySet(elem.evaluate() for elem in b)
             )
             return PrettySet(a_set.union(b_set))
+        
+        def set_difference(a, b):
+            """Represents a set difference."""
+            a_set = (
+                a
+                if isinstance(a, PrettySet)
+                else PrettySet(elem.evaluate() for elem in a)
+            )
+            b_set = (
+                b
+                if isinstance(b, PrettySet)
+                else PrettySet(elem.evaluate() for elem in b)
+            )
+            return PrettySet(a_set.difference(b_set))
+
+        def set_symmetric_difference(a, b):
+            """Represents a set symmetric difference."""
+            a_set = (
+                a
+                if isinstance(a, PrettySet)
+                else PrettySet(elem.evaluate() for elem in a)
+            )
+            b_set = (
+                b
+                if isinstance(b, PrettySet)
+                else PrettySet(elem.evaluate() for elem in b)
+            )
+            return PrettySet(a_set.symmetric_difference(b_set))
+
+        # fmt: on
 
         class SetBinaryOp(BinaryNode):
             opns_map = {
                 "∩": set_intersection,
-                "∪": set_union,
                 "&": set_intersection,
+                "∪": set_union,
                 "|": set_union,
+                "-": set_difference,
+                "−": set_difference,
+                "^": set_symmetric_difference
```

Comment:
A minor point, but a style I prefer: since Python accepts trailing ',' (comma) on lists, tuples, and dicts, whenever I have one that uses 1 line per element, I always end my last element with another ','. This way, if I ever add another element in the future, this line doesn't have to be changed just to add the comma. The result is a diff that is cleaner, just containing the newly-added element (which I would again add with its own trailing comma). (It annoys me that JSON does not accept these trailing commas...)

Reply:
Also, I didn't mind the ∆ operator, I just wanted to also add the ASCII synonym '^', which you did.